src/esp/bindings_js/index.js

```diff
@@ -5,17 +5,25 @@ import VRDemo from "./modules/vr_demo";
 import "./bindings.css";
 
 Module.preRun.push(() => {
-  let scene = "skokloster-castle.glb";
+  let config = {};
+  config.scene = "skokloster-castle.glb";
```

Comment:
Also, use defaults.js file here to export and load defaults (forgot in last PR).